### DISCUSSION TITLE
Is there a way to sync storage buckets and policies in between cloud and local instances?

### DISCUSSION THREAD

**Discussion by sangonzal** [supabase/supabase#13184] [Questions]
I created a bucket with various policies via the portal for cloud instance. I want to recreate that setup locally. Is there a way to sync these? Something along the lines of the "supabase db remote commit" command, but for storage.

**Comment by james-benns** [comment]
I'm also wondering the same thing. "supabase db remote commit" is pulling the policies, but the buckets aren't created. It works if I manually create the buckets through the local dashboard, but that would have to be done each time I start Supabase! Anyone got a solution?

**Reply by james-benns** [reply]
I solved my problem by populating the storage.buckets table in seed.sql script. This creates the buckets every time Supabase starts.

**Comment by matteohoeren** [comment]
https://gist.github.com/matteohoeren/d81eabde6b83420a194ded6202f006f8

I've created a script to do just that for me automatically. The Buckets and their policies should already exist, for me that was the case with `supabase db pull`
